patterns/about-2.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Title: About 2
+ * Slug: twentytwentyfour/about
+ * Categories: about, featured, text
```

Comment:
I agree we can decide on the naming later but now we have 2 patterns with the same slug. Also, about is not a default category, is it?

Reply:
I'll change the slug to match the name (about-2). About is a default category; although we don't use it much (yet).

Reply:
TIL, thank you!